### PR TITLE
when HomePage initial state username is not null, submit the form to …

### PR DESCRIPTION
**Pull request by suhaotian**
…load repos

### CODE REVIEW

**Comment by mxstbr** [comment]
When does this happen?

**Comment by suhaotian** [comment]
@mxstbr 
**app/containers/HomePage/reducer.js**
```js
// The initial state of the App
const initialState = fromJS({
  username: 'mxstbr',
});
```

when username have a default value and not null, reload page doesn't load repos.

**Comment by suhaotian** [comment]
Maybe we can rewrite it like **redux-saga/examples/async** code

File: app/containers/HomePage/index.js [hunk 1]
```diff
@@ -123,7 +132,8 @@ function mapDispatchToProps(dispatch) {
     onChangeUsername: (evt) => dispatch(changeUsername(evt.target.value)),
     changeRoute: (url) => dispatch(push(url)),
     onSubmitForm: (evt) => {
-      evt.preventDefault();
+      /* eslint no-unused-expressions: ["error", { "allowShortCircuit": true }] */
```
Comment (mxstbr) [inline app/containers/HomePage/index.js]: Change this to `// eslint-disable-line no-unused-expressions`!

**Comment by mxstbr** [comment]
Ah, good catch, I like this solution! :+1: One small nitpick, then we can merge this fix—thanks so much!

**Comment by suhaotian** [comment]
change it, but **npm run lint** not pass:
```bash
  136:7  error  Expected an assignment or function call and instead saw an expression  no-unused-expressions

✖ 1 problem (1 error, 0 warnings)
```

**Comment by mxstbr** [comment]
Yeah, I think it needs to be on the same line, but this looks fine. Thanks @suhaotian!

**Comment by suhaotian** [comment]
@mxstbr My pleasure!

**Pull request merged**